packages/expect/src/asymmetricMatchers.ts

```diff
@@ -13,7 +12,7 @@ import {emptyObject} from './utils';
 
 export class AsymmetricMatcher {
   $$typeof: Symbol;
-  inverse: boolean;
+  inverse?: boolean;
```

Comment:
This is needed so that we don't have to re-declare this property the matchers inheriting `AsymmetricMatcher`